resolvers/company.resolvers.go

```diff
@@ -0,0 +1,59 @@
+package resolvers
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dotkom/a3s/ent/company"
+	"github.com/dotkom/a3s/graph/generated"
+	"github.com/dotkom/a3s/graph/model"
+)
+
+func (r *queryResolver) Companies(ctx context.Context) ([]*model.Company, error) {
+	background := context.Background()
+	all, err := r.Client.Company.Query().All(background)
+	if err != nil {
+		return nil, err
+	}
+	var list []*model.Company
+	for _, result := range all {
+		record := model.Company{
+			ID:               fmt.Sprintf("%d", result.ID),
+			Name:             result.Name,
+			ShortDescription: result.ShortDescription,
+			LongDescription:  result.LongDescription,
+			ImageURL:         result.ImageURL,
+			Site:             &result.Site,
+			ContactEmail:     &result.ContactEmail,
+			ContactTel:       &result.ContactTel,
+		}
+		list = append(list, &record)
+	}
```

Comment:
Wondering if there is a prettier and less repetitive way of doing this...

Reply:
Should be a clean way to do this with Repository pattern and Generics (soon)